cpp/modmesh/view/R3DWidget.hpp

```diff
@@ -73,7 +71,7 @@ class RScene
 
 private:
 
-    Qt3DExtras::QAbstractCameraController * m_controller = nullptr;
+    CameraController * m_controller;
```

Comment:
small refactoring to `controller()` getters. Logic didn't change, but because our `CameraController` class is used more often than Qt's less `dynamic_cast`'s will happen